recipes/shtns/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "shtns" %}
+{% set version = "3.3.1" %}
+{% set revision = "r694" %}
```

Comment:
We generally only build released versions of code

Reply:
Perhaps the terminology is wrong (i.e. calling it a revision), but this is a released version of code: 3.3.1. It's just that the download has a suffix (in this case r694).

Reply:
OK